content/vue.md

```diff
@@ -34,7 +34,7 @@ Vue has an excellent [guide and documentation](https://vuejs.org/v2/guide/). Thi
 
 Vue is a frontend library, like React, Blaze and Angular. 
 
-Some really nice frameworks are built around Vue. [Nuxt.js](https://nuxtjs.org) for example, aims to create a framework flexible enough that you can use it as a main project base or in addition to your current project based on Node.js.  Though Nuxt.js is full-stack and very pluggable. It lacks the an API to communicate data from and to the server. Also unlike Meteor, Nuxt still relies on a configuration file. 
+Some really nice frameworks are built around Vue. [Nuxt.js](https://nuxtjs.org) for example, aims to create a framework flexible enough that you can use it as a main project base or in addition to your current project based on Node.js. Though Nuxt.js is full-stack and very pluggable, it lacks an API to communicate data from and to the server. Also unlike Meteor, Nuxt still relies on a configuration file. 
```

Comment:
Here I'm not sure I translated the idea properly. I guess the author meant something between `the` and `an` so I may need some guidance here 🙂